api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTransferInList.java

```diff
@@ -146,7 +146,7 @@ public ReportDefinition constructReportDefinition() {
 
     @Override
     public String getVersion() {
-        return "0.1";
+        return "0.2";
```

Comment:
@ssmusoke  why is this hardcoded

Reply:
@slubwama All reports have versions which allows updates when the versions change

Reply:
@ssmusoke don't you think there is a better way to have this other than the hardcoding